Rakefile

```diff
@@ -185,3 +185,16 @@ namespace :tf do
     end
   end
 end
+
+namespace :docs do
+  desc 'Prints markdown links for resource doc files to update the README'
+  task :resource_links do
+    files = Dir.entries('docs/resources').select { |f| !File.directory? f }
+
+    puts "\n"
+    files.each do |file|
+      puts "- [#{file.split('.')[0]}](docs/resources/#{file})"
+    end
+    puts "\n"
```

Comment:
Might want to incorporate the whole thing in chained list processing instead of using local assignment for part of it.
```ruby
task :resource_links do
  puts "\n"
  Dir.entries('docs/resources')
     .select { |file| !File.directory?(file) }
     .collect { |file| "- [#{file.split('.')[0]}](docs/resources/#{file})" }
     .map { |link| puts link }
  puts "\n"
end
```

Reply:
Awesome idea, updated!